include/phasar/DataFlow/IfdsIde/InitialSeeds.h

```diff
@@ -72,39 +72,6 @@ template <typename N, typename D, typename L> class InitialSeeds {
   [[nodiscard]] const GeneralizedSeeds &getSeeds() const & { return Seeds; }
   [[nodiscard]] GeneralizedSeeds getSeeds() && { return std::move(Seeds); }
 
-  void dump(llvm::raw_ostream &OS = llvm::errs()) {
-
-    auto printNode = [&](auto &&Node) { // NOLINT
-      if constexpr (std::is_same_v<const llvm::Instruction *, N>) {
-        OS << llvmIRToString(Node);
-      } else {
-        OS << Node;
-      }
-    };
-
-    auto printFact = [&](auto &&Node) { // NOLINT
-      if constexpr (std::is_same_v<const llvm::Value *, D>) {
-        OS << llvmIRToString(Node);
-      } else {
-        OS << Node;
-      }
-    };
-
-    OS << "======================== Initial Seeds ========================\n";
-    for (const auto &[Node, Facts] : Seeds) {
-      OS << "At ";
-      printNode(Node);
-      OS << "\n";
-      for (const auto &[Fact, Value] : Facts) {
-        OS << "> ";
-        printFact(Fact);
-        OS << " --> \\." << Value << "\n";
-      }
-      OS << "\n";
-    }
-    OS << "========================== End Seeds ==========================\n";
-  }
-
```

Comment:
not sure whether I would remove this even if dead code, might come handy for debugging purposes?